current/getting-started/glossary.md

```diff
@@ -1,4 +1,6 @@
 ---
+slug: codec
+lang: en
 title: "Glossary"
```

Comment:
Not sure it matters, but this one has quotes, and others don't

Reply:
Ah, thought I deleted them all!